tests/snuba/api/endpoints/test_organization_transaction_details.py

```diff
@@ -0,0 +1,406 @@
+            self.url,
+            data={
+                "start_timestamp": str(datetime.now().timestamp()),
+                "end_timestamp": str(datetime.now().timestamp()),
+            },
+            format="json",
+        )
+        assert response.status_code == 404, response.content
+
+    def test_returns_spans_without_being_given_timestamps(self):
+        self.login_as(user=self.owner)
+
+        start_ts = datetime.now() - timedelta(minutes=1)
+        span = self.create_span(
+            organization=self.organization,
+            project=self.project,
+            start_ts=start_ts,
+            extra_data={
+                "trace_id": self.trace_id,
+                "event_id": self.transaction_id,
+                "is_segment": True,
+            },
+        )
+        self.store_span(span)
+
+        response = self.client.get(
+            self.url,
+            format="json",
+        )
+        assert response.status_code == 200, response.content
+        assert response.data["contexts"]["trace"]["span_id"].endswith(span["span_id"])
+
+    def test_imitates_the_organization_event_details_endpoint_format(self):
+        self.login_as(user=self.owner)
+
+        release = "backend@123"
+        transaction_name = "/api/0/{organization_slug}/{project_slug}/transactions/{transaction_id}"
+        environment = "prod"
+        transaction_op = "http.server"
+        span_duration = 1000
+        browser_name = "Chrome"
+        os_name = "Linux"
+        sdk_name = "javascript"
+        sdk_version = "8.0.0"
+        span_hash = "abc123"
+        start_ts = datetime.now() - timedelta(minutes=1)
+
+        root_span = self.create_span(
+            organization=self.organization,
+            project=self.project,
+            start_ts=start_ts,
+            duration=span_duration,
+            extra_data={
+                "trace_id": self.trace_id,
+                "transaction_id": self.transaction_id,
+                "segment_id": self.transaction_id,
+                "event_id": self.transaction_id,
+                "parent_span_id": None,
+                "is_segment": True,
+                "op": transaction_op,
+                "sentry_tags": {
+                    "transaction": transaction_name,
+                    "release": release,
+                    "environment": environment,
+                    "op": transaction_op,
+                    "browser.name": browser_name,
+                    "os.name": os_name,
+                    "sdk.name": sdk_name,
+                    "sdk.version": sdk_version,
+                    "group": span_hash,
+                },
+                "tags": {
+                    "tag": "value",
+                },
+            },
+        )
+        self.store_span(root_span)
+
+        response = self.client.get(
+            self.url,
+            format="json",
+        )
+        assert response.status_code == 200, response.content
+        assert response.data == {
```

Comment:
🤔 this seems like it'll be a pain to maintain, thoughts on using something like `src/sentry/data/samples/transaction.json` so that most of these fields are static?